components/authz-service/storage/postgres/migration/migration.go

```diff
@@ -63,9 +63,10 @@ func (c *Config) Migrate(dataMigConf datamigration.Config) error {
 	}
 
 	if version < PreForceUpgradeMigration {
+		l.Infof("Migrating schema version %v to %v...", version, PreForceUpgradeMigration)
 		err = m.Migrate(PreForceUpgradeMigration)
 		if err != nil && err != migrate.ErrNoChange {
-			return errors.Wrap(err, "migration up to IAM-V2-force-upgrade failed")
+			return errors.Wrapf(err, "migration up to version %v failed", PreForceUpgradeMigration)
```

Comment:
Review sentiment seemed to favor removing v1 and v2 references, so updated a few error messages in that regard.